tests/ci/cdk/cdk/codebuild/github_ci_linux_arm_omnibus.yaml

```
@@ -289,10 +289,18 @@ batch:
         variables:
           AWS_LC_SSL_RUNNER_START_INDEX: 7001
 
-#   - identifier: amazonlinux2023_gcc11x_aarch_valgrind
-#     buildspec: ./tests/ci/codebuild/common/run_valgrind_tests.yml
-#     env:
-#       type: ARM_CONTAINER
-#       privileged-mode: true
-#       compute-type: BUILD_GENERAL1_LARGE
-#       image: 620771051181.dkr.ecr.us-west-2.amazonaws.com/aws-lc-docker-images-linux-aarch:amazonlinux-2023_gcc-11x_latest
+    - identifier: amazonlinux2023_gcc11x_aarch_valgrind
+      buildspec: ./tests/ci/codebuild/common/run_valgrind_tests.yml
+      env:
+        type: ARM_CONTAINER
+        privileged-mode: true
+        compute-type: BUILD_GENERAL1_LARGE
+        image: 620771051181.dkr.ecr.us-west-2.amazonaws.com/aws-lc-docker-images-linux-aarch:amazonlinux-2023_gcc-11x_latest
+
+    - identifier: amazonlinux2023_gcc11x_aarch_ssl_valgrind
+      buildspec: ./tests/ci/codebuild/common/run_ssl_valgrind_tests.yml
```

```suggestion
    - identifier: amazonlinux2023_gcc11x_aarch_ssl_runner_valgrind
      buildspec: ./tests/ci/codebuild/common/run_ssl_runner_valgrind_tests.yml
```
Same comments here about clarifying this is just for the ssl runner tests.

---

Done.